README.md

```diff
@@ -153,9 +153,114 @@ This library doesn't handle this for you, but here are the 2 main options:
+
+First, in your form type:
+ - You need to add an attribute on the choice field, so it auto-submits the form when changed (may need to be adapted to your own form if more complex)
+ - Add a submit button, so in the controller you can differenciate from an auto-submit versus a user action
+
+
+```diff
+// src/Form/FeedbackForm.php
+
+// ...
+
+class FeedbackForm extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder =  new DynamicFormBuilder($builder);
+
+        $builder->add('rating', ChoiceType::class, [
+            'choices' => [
+                'Select a rating' => null,
+                'Great' => 5,
+                'Good' => 4,
+                'Okay' => 3,
+                'Bad' => 2,
+                'Terrible' => 1
+            ],
++           // This will allow the form to auto-submit on value change
++           'attr' => ['onchange' => 'this.form.submit()'],
+        ]);
++       // This will allow to differenciate between a user submition and an auto-submit
++       $builder->add('submit', SubmitType::class, [
++           'attr' => ['value' => 'submit'], // Needed for Turbo
++       ]);
+
+        $builder->addDependent('badRatingNotes', 'rating', function(DependentField $field, ?int $rating) {
+            if (null === $rating || $rating >= 3) {
+                return; // field not needed
+            }
+
+            $field->add(TextareaType::class, [
+                'label' => 'What went wrong?',
+                'attr' => ['rows' => 3],
+                'help' => sprintf('Because you gave a %d rating, we\'d love to know what went wrong.', $rating),
+            ]);
+        });
+    }
+}
+```
+
+Second, in your controller:
+ - Specify the action on your form, [this is needed for Turbo Frame](https://symfony.com/bundles/ux-turbo/current/index.html#3-form-response-code-changes)
+ - Handle the auto-submit by checking if the button has been clicked
+
+```diff
+// src/Controller/FeedbackController.php
+
+    #[Route('/feedback', name: 'feedback')]
+    public function feedback(Request $request): Response
+    {
+        //...
+
+-       $feedbackForm = $this->createForm(FeedbackForm::class);
++       $feedbackForm = $this->createForm(FeedbackForm::class, options: [
++           // This is needed by Turbo Frame, it is not specific to Dependent Symfony Form Fields
++           'action' => $this->generateUrl('feedback'),
++       ]);
+        $feedbackForm->handleRequest($request);
+        if ($feedbackForm->isSubmitted() && $feedbackForm->isValid()) {
+
++           /** @var SubmitButton $submitButton */
++           $submitButton = $feedbackForm->get('submit');
++           if (!$submitButton->isClicked()) {
++               return $this->render('feedback.html.twig', ['feedbackForm' => $feedbackForm]);
++           }
+
+            // Your code here
+            // ...
+
+            return $this->redirectToRoute('home');
```

Comment:
And if so, maybe better to do the opposite.... if `$submitButton->isClicked()` is clicked - do this `return $this->redirectToRoute('home');`.

Otherwise, the execution will continue and call that `return $this->render('feedback.html.twig', ['feedbackForm' => $feedbackForm]);` below.

Wdyt?